### DISCUSSION TITLE
StableStudio vs StableSwarmUI

### DISCUSSION THREAD

**Discussion by metamas** [Stability-AI/StableStudio#106] [Q&A]
What are the major differences in features and roadmap between StableSwarmUI and StableStudio? In short time, I've hopped from A1111 to ComfyUI to StableStudio... and now I'm looking at StableSwarmUI and thinking "Does this one do everything all the others do and more?". It's all a bit disorienting.

As StableSwarmUI and StableStudio are both projects with StabilityAI, I'm curious if one will be abandoned in favor of the other, or if both are intended to be developed in parallel and serve different audiences?

**Comment by southbayjay** [comment]
Seems like this project is already been left for dead.. Look at the other topics that have gone unanswered. I was looking to install this to give it a try but it doesn't seem like there's much support. I'd suggest jumping over to StableSwarm which has much more users and better support.  Even the disco server for StableStudio is dead, meanwhile the Stable Swarm disco is very active.
